tests/turtle/eval/manifest.ttl

```diff
@@ -0,0 +1,79 @@
+## Distributed under both the "W3C Test Suite License" [1]
+## and the "W3C 3-clause BSD License".
+## [1] https://www.w3.org/Consortium/Legal/2008/04-testsuite-license
+## [2] https://www.w3.org/Consortium/Legal/2008/03-bsd-license
+
+PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#>
+PREFIX rdft:   <http://www.w3.org/ns/rdftest#>
+PREFIX :       <#>
+
+<>  rdf:type mf:Manifest ;
+    rdfs:label "Turtle* EvaluationSyntax" ;
```

Comment:
```suggestion`
    rdfs:label "Turtle* Evaluation Tests" ;
```

Reply:
Changes made.  Thanks for catching the errors.